src/functTest/groovy/com/bmuschko/gradle/docker/tasks/image/DockerfileFunctionalTest.groovy

```diff
@@ -156,6 +157,42 @@ MAINTAINER Benjamin Muschko "benjamin.muschko@gmail.com"
 """
     }
 
+    def "Can create Dockerfile from template file"() {
+        given:
+        buildFile << """
+import com.bmuschko.gradle.docker.tasks.image.Dockerfile
+
+task ${DOCKERFILE_TASK_NAME}(type: Dockerfile) {
+    instructionsFromTemplate 'src/main/docker/Dockerfile.template'
+}
+"""
+        when:
+        build(DOCKERFILE_TASK_NAME)
+
+        then:
+        File dockerfile = new File(projectDir, 'build/docker/Dockerfile')
+        dockerfile.exists()
+        dockerfile.text ==
```

Comment:
This complex template makes this test less readable. I think it can be as simple as:
````Dockerfile
FROM ubuntu:14.04
MAINTAINER Benjamin Muschko "benjamin.muschko@gmail.com
````